src/testing/controller/scan/export/controller.go

```diff
@@ -0,0 +1,78 @@
+package export
```

Comment:
This file seems not generated by mockery?

Reply:
Done. Cleaned up